.gitignore

```diff
@@ -27,3 +27,5 @@ dist/
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules/
 bower_components/
+
+**/*.DS_Store
```

Comment:
not sure we need to have this... think you can just delete it from your local repo... think it's just a weird OSX thing?